geode-core/src/main/java/org/apache/geode/cache/client/Pool.java

```diff
@@ -140,7 +140,10 @@
    * Returns <code>true</code> if thread local connections are enabled on this pool.
    *
    * @see PoolFactory#setThreadLocalConnections
+   * @deprecated Since Geode 1.10.0. Thread local connections are ignored. Will be removed in future
+   *             major release. Now always returns false.
    */
+  @Deprecated
```

Comment:
something I think you could do here, that would allow you to delete all the impls of this method, is mark it here as "default" and just have it return false here as the default implementation.

Reply:
Hmmmmm interesting idea. Because we are having this message log a warning it would require that the interface gain a logger. Not sure I’d like that. I guess the method could new a logger itself. Think it’s worth it?

Reply:
No. You are right that it could just create the logger but I think it would be better to leave like you have it now.